src/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+const App: React.FC = () => {
+  return <h1>Hey there champ!! How are you doing?</h1>;
+};
```

Comment:
Would you create a separate `App.tsx` file to hold this logic?

Having the entry and the component logic in the file file is not a good practice!